src/containers/Settings/sagas.js

```diff
@@ -0,0 +1,19 @@
+import request from 'utils/request';
+import { push } from 'react-router-redux';
+import { put, takeLatest } from 'redux-saga/effects';
+import { actionTypes as bountyActionTypes } from 'public-modules/Bounty';
+
+const { CREATE_DRAFT_SUCCESS, CREATE_BOUNTY_SUCCESS } = bountyActionTypes;
```

Comment:
haha copying and pasting?  So confused by what this is.

Reply:
yeah 😞